free-flow-over-porous-media/compile-dumux-cases.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env sh
+set -e -u
+
+cd free-flow-dumux/build-cmake/appl
+make free_flow_dumux
+cd ../../../porous-media-dumux/build-cmake/appl
+make porous_media_dumux
+cd ../../../
```

Comment:
I would write this as:

```suggestion
(
  cd free-flow-dumux/build-cmake/appl
  make free_flow_dumux
)

(
  cd porous-media-dumux/build-cmake/appl
  make porous_media_dumux
)
```

Easier to read and the two steps don't depend on each other.